test/unit/mdc-list/mdc-list.test.js

```diff
@@ -317,19 +301,76 @@ test('selectedIndex calls setSelectedIndex on foundation', () => {
   td.verify(mockFoundation.setSelectedIndex(1), {times: 1});
 });
 
+test('focusIn handler is added to root element', () => {
+  const {root, mockFoundation} = setupTest();
+  document.body.appendChild(root);
+  const event = document.createEvent('Event');
```

Comment:
You can use the `dom-events` module to make all of these tests more concise.

Reply:
I tried updating to `dom-events` but ran into issues with the focus methods bubbling. I also found that `dom-events` create a `KeyboardEvent` instead of just an `Event` and that may cause issues.